<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Recupera Password - EVENTICA</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #1E90FF;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>750</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
qproperty-alignment: AlignCenter;</string>
    </property>
    <property name="text">
     <string>EVENTICA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="email_input">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>220</y>
      <width>271</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
color: #1E90FF;
padding: 5px;
border-radius: 3px;</string>
    </property>
    <property name="placeholderText">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QPushButton" name="recupera_button">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>270</y>
      <width>271</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: white;
   color: #1E90FF;
   font-weight: bold;
   border-radius: 5px;
   padding: 8px;
}
QPushButton:hover {
   background-color: #F0F0F0;
}</string>
    </property>
    <property name="text">
     <string>Recupera Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="error_label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>320</y>
      <width>271</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: bold;
qproperty-alignment: AlignCenter;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="back_button">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>360</y>
      <width>271</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   background-color: white;
   color: #1E90FF;
   font-weight: bold;
   border-radius: 5px;
   padding: 8px;
}
QPushButton:hover {
   background-color: #F0F0F0;
}</string>
    </property>
    <property name="text">
     <string>Ritorna al Login</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>42</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>